student.rb

```diff
@@ -4,8 +4,8 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(classroom, age, name, parent_permission)
-    super()
+  def initialize(classroom)
+    super(age, name, parent_permission)
```

Comment:
- [ ] currently, your add student is not working well this could be because of your student.rb initialize, you can make your initialize take three parameters like `classroom, age, name = 'Unkown'`
the error :point_down: 
![Screenshot from 2022-05-31 13-26-46](https://user-images.githubusercontent.com/66097371/171173763-866cd9e4-a043-4e92-98ca-1a50f6b49942.png)